### PR TITLE
Fix Account Block Alert Issue

### PR DESCRIPTION
**Pull request by HeyMehedi**
## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Security fix
- [x] Improvement
- [ ] New Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Text changes
- [ ] Other... Please describe:

## Description

New user registration is not working properly especially if you make a mistake during registration. 
the window closes by itself, etc. "goes into error mode"

**Before:** https://www.loom.com/share/d70eb2e06e2348188fde45f51f2d5a28
**After:** https://www.loom.com/share/a58486ef5b8c480b9a01fcfbf057a04e

**Task:**
Authentication Form Reset on Form Change
Form Submit event changes to onClick by Button

## Any linked issues

The form is not resetting after getting an error
The submit button does not properly work in Block

Taiga issue: **483**

## Checklist

- [x] My code follows the [WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)

### CODE REVIEW

**Comment by vairafiq** [comment]
PR description is missing! Please add

1. Way to reproduce
2. Before and after screenshots

**Comment by HeyMehedi** [comment]
> PR description is missing! Please add
> 
> 1. Way to reproduce
> 2. Before and after screenshots

Updated the description, please check.

File: templates/account/login-registration-form.php [hunk 1]
```diff
@@ -108,7 +108,7 @@
 							}
 						}
 					} else { ?>
```
Comment (vairafiq) [inline templates/account/login-registration-form.php:110]: Need to change the template version

**Review by vairafiq — CHANGES_REQUESTED** [review]

File: templates/account/login-registration-form.php (hunk 1 above)
Comment (HeyMehedi) [reply templates/account/login-registration-form.php:110]: updated

**Review by vairafiq — APPROVED** [review]

**Pull request merged**